indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -139,6 +142,7 @@
                         url: url,
                         method: method,
                         data: params,
+                        content: $this.parent().find(content).html(),
```

Comment:
Is going to the parent strictly necessary? If we **always** expect `content` to be descendant of `$this` we can simply use `$this.find()`. If we **never** expect `content` to be descendant of `this` we should simply use `.siblings()`. If we can't make any of these assumptions then I said nothing. :P